src/test/java/tools/jackson/databind/jsontype/ext/ExternalTypeIdTest.java

```diff
@@ -616,7 +616,7 @@ public Orange(@JsonProperty("name") String name, @JsonProperty("name") String co
     public void testIssue3008() throws Exception
     {
         ObjectReader r = MAPPER.readerFor(Box3008.class);
-        Box3008 deserOrangeBox = r.readValue("{\"type\":null,\"fruit\":null}}");
+        Box3008 deserOrangeBox = r.readValue("{\"type\":null,\"fruit\":null}");
```

Comment:
The test is not about JSON Syntax error. So just remove additional closing bracket.